userguide/content/en/docs/Updating/Convert-site-to-module.md

```diff
@@ -171,32 +168,47 @@ hugo: cleaned module cache for "github.com/twbs/bootstrap"
 ```
 {{% /alert %}}
 
-### Clean up your repository
+## Clean up of your repository
```

Comment:
I would drop "of" here. (@LisaFC WDYT?)

Reply:
Yes, leave out the "of" - just "Clean up your repository" works

Reply:
Changed as requested.